client/state/country-states/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function getCountryStates( state, countryCode ) {
```

Comment:
I know these are pretty simple, but do you mind adding some jsdoc and tests for the selectors?